src/screens/CreatePost/index.js

```diff
@@ -187,9 +188,6 @@
 
   React.useEffect(() => {
     init();
-    if (isInCreatePostTopicScreen) {
-      setTimeout(() => onChangeText(`${message}#${listTopic[0]}`), 500);
-    }
   }, []);
 
   const init = async () => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `init` function is now being called inside a `useEffect` hook with an empty dependency array, which means it will only be called once when the component mounts. This is a good practice for initialization logic that should only run once.
```diff
+ React.useEffect(() => {
+   init();
+ }, []);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->